### PR TITLE
Card Class and Deck Class

### PR DESCRIPTION
**Pull request by pvlnguyen**
Draw function is currently unavailable

### CODE REVIEW

File: deck.py.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+VALUES = ["A", "2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q" , "K"]
```
Comment (MoeyyadQureshi) [inline deck.py.py]: nitpick: delete extra space after "Q"

File: deck.py.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+SUITS = ["s", "c", "d", "h"]
+
+import random
```
Comment (MoeyyadQureshi) [inline deck.py.py]: nitpick: imports should be at the very top of a file

File: deck.py.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        print ("{} of {}".format(self._value, self._suit)
+
+class Deck:
+    
+    def __init__(self):
+        self._cards = []
+        self.build()
+        
+# build is a function that generates a list of 52 cards
+# build: None -> None
```
Comment (MoeyyadQureshi) [inline deck.py.py]: nitpick: In Python 3.4 and after you can do `def build(self) -> None:` instead of having a comment :)

File: deck.py.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    def build(self):
+        for suit in SUITS:
+            for value in VALUES:
+                cards.append(Card(value, suit))
+    
+    def shuffle(self):
+        self.random.shuffle()
```
Comment (MoeyyadQureshi) [inline deck.py.py]: Pass in the list of cards to the shuffle method

File: deck.py.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+import random 
+
+class Card:
+    
+    def __init__(self, value, suit):
+        self._value = value
+        self._suit = suit
+        
+    def show(self):
+        print ("{} of {}".format(self._value, self._suit)
```
Comment (MoeyyadQureshi) [inline deck.py.py]: nitpick: as of Python 3.6 you can do `f"{self._value} of {self._suit}"`

File: deck.py.py (hunk 2 above)
Comment (pvlnguyen) [reply deck.py.py]: Done

File: deck.py.py (hunk 3 above)
Comment (pvlnguyen) [reply deck.py.py]: Done

File: deck.py.py (hunk 5 above)
Comment (pvlnguyen) [reply deck.py.py]: Done

**Pull request closed without merging**